app/assets/javascripts/quby/answers/validation.js

```diff
@@ -189,6 +208,48 @@
     return !failed;
   };
 
+  function parsePartialDate(inputs) {
+    var value = function(placeholder) {
+      var val = inputs.filter("[placeholder=" + placeholder + "]").first().val();
+      if(val === undefined || val == "")
```

Comment:
Doch, een 'e' is niet leeg, wordt hier teruggegeven als een NaN, (NaN || 2000) == 2000.